docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+services:
+  task_db:
+    image: postgres
+  task:
+    build: .
+    command: bash -c "python manage.py migrate && python manage.py runserver 0.0.0.0:8001"
```

Comment:
@awm33 : yea - as in that issue. We may want to have both APIs hosted the same way. Particularly if a number of workers are going to access this one.